DEFAULT_CONFIG.json5

```diff
@@ -261,6 +261,13 @@
             data_low: 4,
             background: 4,
           },
+          /// Congestion occurs when the queue is empty (no available batch).
+          /// Using CongestionControl::Block the caller is blocked until a batch is available and re-insterted into the queue.
+          /// Using CongestionControl::Drop the message might be dropped, depending on conditions configured here.
+          congestion_control: {
+            /// The maximum time in nanosecond to wait for an available batch before dropping the message if still no batch is available.
```

Comment:
Should it be in microsends? I believe it's more user friendly and I don't really see the need to expose such granurality that may not even be guaranteed by the system timer.

Reply:
I choose nanoseconds to be consistent with the `backoff` setting which is in nanoseconds.
From a user perspective, I feel weird and possibly confusing that different time units have to be used in a same config file.

Reply:
We already have different time units depending on the configuration variable and on the context, e.g.: https://github.com/eclipse-zenoh/zenoh/blob/5f6037293a6c425d5769797010257271924228c6/DEFAULT_CONFIG.json5#L233

Reply:
Fair enough, I changed the unit to microseconds